connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectService.scala

```diff
@@ -175,6 +229,8 @@ object SparkConnectService extends Logging {
 
   private val CACHE_TIMEOUT_SECONDS = 3600
 
+  private val PROTOBUF_RECURSION_LIMIT = 1024
```

Comment:
Maybe add this to Config. Could come handy to be able to change it if it turns out to be too low / too high.